src/parsers/md.py

```diff
@@ -21,6 +21,19 @@ def get_meta(question_dict: dict, key: str):
         return question_dict["metadata"][key][0]
     return question_dict["metadata"][key]
 
+def set_meta(question_dict: dict, key: str, value: any):
+    """
+    Set metadata in question_dict
+    :param question_dict: dictionary that contains question metadata
+    :param key: the key that will be searched in the dictionary
+    :param value: the value that will be set in the dictionary
+    """
+    if key in question_dict["metadata"]:
+        question_dict["metadata"][key].append(value)
+    else:
+        question_dict["metadata"][key] = []
+        question_dict["metadata"][key].append(value)
```

Comment:
```suggestion
    if key not in question_dict["metadata"]:
        question_dict["metadata"][key] = []
    question_dict["metadata"][key].append(value)
```
`else` is overrated.